src/components/GamePage.vue

```diff
@@ -4,6 +4,7 @@ import { ref, onMounted, Vue, reactive} from "vue"
 import { codeStore } from '../stores/code'
 
 const code = codeStore();
+const { setNextIndexCode, controlIndex} = code;
```

Comment:
```suggestion
const { setNextIndexCode, controlIndex} = storeToRefs(code);
```
今朝、お話したstoreToRefs()をつけたほうがよいと思います。
setNextIndexCodeを定義していますが、GamePage.vueの中で使われていないようなので削除していいと思いました。

Reply:
storeToRefsは変更してみましたが動きませんでした。

Reply:
storetorefsは変数をリアクティブに保つものであるらしいです。この場合関数なので必要ないかと思われます。

Reply:
そうでした！actionsないのメソッドには必要ありませんでした。申し訳ないです！